test/functional/p2p_ibd_txrelay.py

```diff
@@ -6,8 +6,27 @@
 
```

Comment:
The docstring in this file could be updated. It's no longer just testing fee filters.

Reply:
Yes updates the docstring, lmk about the formatting by bullet point.